package/src/specs/RNCandle.nitro.ts

```diff
@@ -6,6 +6,225 @@ export type AppUser = {
   appUserID?: string;
 };
 
+type ACHAccountKind = "checking" | "savings";
+
+export type ACHDetails = {
+  accountNumber: string;
+  routingNumber: string;
+  accountKind: ACHAccountKind;
+};
+
+export type WireDetails = {
+  accountNumber: string;
+  routingNumber: string;
+};
+
+type FiatAccountDetails = {
+  assetKind: string; // fiat
+  serviceAccountID: string;
+  currencyCode: string;
+  balance?: number;
+  ach?: ACHDetails;
+  wire?: WireDetails;
+  linkedAccountID: string;
+  service: Service;
+};
+
+type MarketAccountDetails = {
+  assetKind: string; // "stock" | "crypto"
+  serviceAccountID: string;
+  linkedAccountID: string;
+  service: Service;
+};
+
+type AssetAccountDetails = {
+  fiatAccountDetails?: FiatAccountDetails;
+  marketAccountDetails?: MarketAccountDetails;
+};
+
+export type LegalAccountKind =
+  | "individual"
+  | "joint"
+  | "traditionalIra"
+  | "rothIra";
```

Comment:
```suggestion
  | "traditional_ira"
  | "roth_ira";
```
This casing is weird for JS